drivers/pds/api/defaultTemplates.go

```diff
@@ -14,14 +14,14 @@ type DefaultTemplates struct {
 }
 
 // ListApplicationConfigurationTemplates func
```

Comment:
I know you didn't write this function, but since you are touching it and it has a comment, can we elaborate more on what this function is doing instead of just `func`, for example:
```
// ListApplicationConfigurationTemplates takes tenant ID and returns application configuration templates for the given tenant
```

Reply:
Done